site/gatsby-site/netlify.toml

```diff
@@ -14,3 +14,6 @@ TERM = "xterm"
     enable = true
     spa = true
     start = 'gatsby serve -p 8080'
+
+[functions]
+  included_files = ["!.cache/data/datastore/*"]
```

Comment:
We are dealing with this by setting: `GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE=1`, please remove these lines.